backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/cgen/CBridgeGen.kt

```diff
@@ -750,6 +752,9 @@ private fun KotlinStubs.mapBlockType(
 private fun KotlinStubs.mapType(type: IrType, retained: Boolean, variadic: Boolean, location: TypeLocation): ValuePassing =
         mapType(type, retained, variadic, location, { reportUnsupportedType(it, type, location) })
 
+private fun IrType.isNullLiteral(): Boolean = this is IrSimpleType && hasQuestionMark
```

Comment:
This function is misnamed: type can't be null literal, because null literal is an expression (`null`).

Reply:
Provide the suggestion please

Reply:
`isNullableNothing` or `isTypeOfNullLiteral`, depending on how exactly you'd like to express the meaning on the call site.